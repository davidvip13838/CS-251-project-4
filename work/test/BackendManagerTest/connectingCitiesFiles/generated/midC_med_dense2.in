500
74 33
0 54
50 73
96 41
6 34
16 3
71 89
45 74
66 58
21 79
100 52
0 86
100 82
4 16
87 99
25 14
54 84
49 87
55 95
49 75
35 4
45 95
13 41
89 2
6 53
10 9
25 87
49 95
99 60
35 80
7 15
54 73
5 92
41 69
79 62
89 97
42 28
43 66
64 78
67 62
78 26
61 7
98 25
56 98
36 69
41 92
64 61
7 63
31 59
97 91
77 61
43 28
19 14
20 86
23 99
92 61
100 3
94 70
80 11
77 44
27 52
15 99
63 31
62 2
35 49
34 30
32 37
10 94
18 17
20 80
34 94
43 68
89 8
52 41
22 3
40 76
73 85
25 74
45 46
13 19
92 45
76 43
7 75
21 4
17 93
67 77
18 59
26 50
51 55
1 0
78 76
54 89
70 96
28 98
22 6
45 13
9 69
46 99
15 40
84 97
54 68
27 38
68 14
92 48
83 13
100 16
80 83
16 67
43 20
54 54
95 80
19 13
47 53
93 67
78 97
86 93
45 11
6 42
97 84
73 62
38 6
16 74
10 24
31 21
56 23
72 96
34 23
8 80
4 19
83 7
40 35
99 79
40 72
24 19
14 93
23 49
8 63
1 68
77 15
85 55
36 42
21 38
89 24
50 25
94 53
62 48
7 19
74 0
50 58
21 8
27 85
0 25
93 15
39 94
25 32
53 9
86 77
68 91
66 17
64 63
88 73
68 94
0 23
28 72
62 46
70 72
62 84
73 42
1 89
56 57
40 94
22 52
93 30
14 94
53 41
48 13
86 79
92 9
100 65
98 22
2 2
36 81
65 76
94 81
25 99
5 25
69 87
74 74
63 68
73 17
81 49
85 69
47 25
40 58
12 20
69 34
79 51
82 20
70 99
68 49
2 47
67 90
60 78
41 28
45 3
77 95
16 96
70 18
37 45
24 98
79 54
2 9
72 6
66 48
94 0
1 25
15 21
80 6
83 26
84 74
65 12
80 4
84 52
75 50
59 73
20 18
76 7
28 5
81 10
9 68
38 88
79 97
13 71
11 40
48 63
25 44
49 15
33 17
98 97
50 77
56 91
65 10
1 20
22 27
87 53
89 55
97 52
35 22
100 14
45 88
29 44
93 86
27 89
36 26
55 57
82 88
16 40
74 53
15 37
74 36
63 32
84 75
96 20
82 22
39 58
24 38
59 30
0 81
85 53
87 16
81 51
21 42
12 28
44 21
10 70
42 82
35 83
57 46
89 6
70 77
55 75
97 20
70 59
51 97
36 45
38 58
54 88
98 14
77 4
89 38
13 86
46 6
88 78
62 28
8 54
62 19
1 91
70 11
45 49
14 89
65 82
90 41
70 63
27 75
40 48
40 69
93 21
41 0
44 68
24 96
96 73
97 10
40 0
55 77
42 19
89 86
2 88
28 29
83 93
86 49
18 46
84 11
94 55
0 73
31 98
95 24
88 42
96 90
80 39
95 8
96 63
78 77
72 49
12 76
73 92
46 16
17 91
73 6
50 43
95 44
35 78
30 65
87 67
32 33
99 84
90 48
39 62
39 80
34 71
0 41
94 89
30 12
12 57
38 94
99 62
88 27
40 98
29 27
58 40
42 98
5 31
44 24
5 30
92 41
83 88
41 5
19 34
39 67
92 58
41 81
82 44
24 40
0 50
5 76
66 71
32 98
55 65
95 97
40 92
44 29
6 88
58 6
92 8
96 75
9 67
22 32
80 84
95 83
21 53
60 31
69 12
97 73
93 100
96 21
24 83
1 65
2 59
63 57
90 31
57 94
64 60
85 5
8 23
42 16
46 0
58 88
6 33
53 43
85 68
31 52
84 16
70 85
88 12
49 1
49 32
31 5
95 5
24 21
84 12
68 51
76 80
100 29
15 30
34 54
11 9
6 89
87 9
90 9
1 23
22 80
5 9
47 50
52 92
4 4
85 83
4 38
47 70
72 4
31 56
66 56
57 24
67 11
18 30
81 28
77 96
43 63
37 81
69 75
65 95
89 40
98 3
68 19
27 21
40 90
36 55
74 95
30 49
100 27
85 75
96 40
20 99
94 49
86 39
40 83
83 36
6 21
92 19
18 90
51 7
20 50
25 46
73 3
23 69
96 69
86 5
99 27
36 15
64 28
21 62
64 39
30 63
71 0
25 20
26 48
40 22
15 94
2 62
27 25
74 83
51 57
79 93
77 35
58 33
53 33
67 47
13 84
29 38
97 24
58 13
93 9
