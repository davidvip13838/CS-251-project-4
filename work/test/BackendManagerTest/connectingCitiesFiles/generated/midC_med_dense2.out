1521.9214132413608